app/src/main/java/fr/free/nrw/commons/contributions/ContributionsContentProvider.java

```diff
@@ -39,11 +41,12 @@ public static Uri uriForId(int id) {
         return Uri.parse(BASE_URI.toString() + "/" + id);
     }
 
-    @Inject DBOpenHelper dbOpenHelper;
+    @Inject
+    Lazy<DBOpenHelper> dbOpenHelper;
```

Comment:
Inconsistent line wrapping compared with other places where annotations are on the same line.

Reply:
Rename for clarity?  See comments on `CategoryContentProvider`

Reply:
Yes, i dont see clear code style conventions being followed across the application. Are you referring to Google style guide lines? I will take care of following it for all my contributions. :)